html/php-pre-challenge3/index.php

```diff
@@ -1,6 +1,80 @@
 <?php
+
+// URLで入力されるtarget値$limitに代入している
 $limit = $_GET['target'];
 
+// もしlimitが１以上の整数でなければ400を返す
+if (preg_match('/^([1-9]\d*|0)\.(\d+)?$/', $limit) || $limit < 1) {
+  http_response_code(400);
```

Comment:
```suggestion
  http_response_code(400);
  $errMsg = 'invalid limit: ' . $limit;
  echo json_encode($errMsg, JSON_UNESCAPED_UNICODE);
  die();
```
http_response_codeではスクリプトは停止しません。dieかexitを書きましょう。
こうするとエラーが画面に出るので親切です。

Reply:
3cc0295 こちらのコミットで実装しました。
65b22e6 2つ前の、こちらのコミットは判定文の移動のみを行っていたので追加実装です